### ISSUE TITLE
package incompatibility with scvi-tools beyond 0.15.0 when running RegressionModel (or tutorial syntax needs update)

### ISSUE THREAD

**Issue by nancyliy** [BayraktarLab/cell2location#149]
- [x] I have confirmed this bug exists on the latest version of cell2location. See https://github.com/BayraktarLab/cell2location#installation
- [x] I follow the instructions from the [scvi-tools tutorial](https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html).

https://docs.scvi-tools.org/en/stable/release_notes/v0.15.0.html

scvi-tools introduced major breaking changes in their API that make the model implementation different (from model agnostic _setanndata to model specific _setanndata). The code in tutorial seems to not have been updated based on this change.
<img width="1211" alt="Screen Shot 2022-05-01 at 10 14 23 PM" src="https://user-images.githubusercontent.com/78202174/166289144-94a6f197-8ffe-4459-b359-1176f4369e4b.png">

---

**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

**Comment by vitkl** [comment]
Hi @nancyliy 

Thanks for using cell2location. The tutorial needs to be updated:
1. Please use `cell2location.models.RegressionModel.setup_anndata(adata_ref, ...)` for reference signature estimation and `cell2location.models.Cell2location.setup_anndata(adata_vis, ...)` for cell2location application to spatial adata_vis.
2. The anndata setup settings can be viewed after the model is created: `cell2location.models.RegressionModel.view_anndata_setup(model)` and `cell2location.models.Cell2location.view_anndata_setup(model)`.

**Comment by katimbach** [comment]
Hi @vitkl,

When I try to modify the script using your recommendation, I get the following error: 
```
    cell2location.models.RegressionModel.view_anndata_setup(model)
AttributeError: type object 'RegressionModel' has no attribute 'view_anndata_setup'
```
I tried to use the adata_ref as the argument instead, but it resulted in the same issue. I have used cell2location in the past and found it very helpful, but I've been having multiple issues since the new pyro version with the tutorial code being buggy and not running as intended. Is there any plan to update the tutorial documentation in the near future?
